28-Github Profile/script.js

```diff
@@ -0,0 +1,101 @@
+const ApiUrl = 'https://api.github.com/users/'
+const form = document.getElementById('form')
+const search = document.getElementById('search')
+const main = document.getElementById('main')
+
+
+async function getUser(username) {
+  try {
+    search.disabled = true
+    const { data } = await axios(ApiUrl + username)
+
+    createUserCard(data)
+    getRepos(username)
+  } catch (err) {
+    if (err.response.status == 404) {
+      createErrorCard('No profile with this username')
```

Comment:
what if there is another error other than the `404`? I think you should also have a generic error message so, for example
```
catch (err) {
    if (err.response.status == 404) createErrorCard('No profile with this username');
    else createErrorCard('Something went wrong, please try again later');
}
```

Reply:
thx